roles/post_install/openstack_tasks/tasks/main.yml

```diff
@@ -0,0 +1,97 @@
+---
+
+- name: Setup Openstack environment
+  tags: setup_os_env
+  import_tasks: setup_openstack_env.yml
+  when: setup_os_env
+
+- name: Instances
+  tags: instance
+  import_tasks: instances.yml
+  when: instance and resource_state == 'absent'
+
+- name: Projects, users and roles creation
+  tags: user
+  import_tasks: users.yml
+  when: user and resource_state == 'present'
+
+- name: Quota
+  tags: quota
+  import_tasks: quota.yml
+  when: quota
+
+- name: Volumes
+  tags: volume
+  import_tasks: volumes.yml
+  when: volume
+
+- name: Security groups creation
+  tags: security_group
+  import_tasks: security_groups.yml
+  when: security_group and resource_state == 'present'
+
+- name: Networks
+  tags: network
+  import_tasks: networks_and_routers.yml
+  when: network
+
+- name: Aggregate groups
+  tags: aggregate
+  import_tasks: aggregate.yml
+  when: aggregate
+
+- name: Flavors
+  tags: flavor
+  import_tasks: flavors.yml
+  when: flavor
+
+- name: Images
+  tags: image
+  import_tasks: images.yml
+  when: image and resource_state == 'present'
+
+- name: Keypair
+  tags: keypair
+  import_tasks: keypair.yml
+  when: keypair
+
+- name: Security groups delete
+  tags: security_group
+  import_tasks: security_groups.yml
+  when: security_group and resource_state == 'absent'
+
+- name: Projects, users and roles creation
+  tags: user
+  import_tasks: users.yml
+  when: user and resource_state == 'absent'
+
+- name: Instances
+  tags: instance
+  import_tasks: instances.yml
+  when: instance and resource_state == 'present'
+
+- name: Generate resources output file
```

Comment:
No resources_output.yml is present

Reply:
A bunch of files are still not present I will add them one by one. This commit is too big already.

Reply:
Please remove this task until the files are present